package.json

```diff
@@ -52,4 +52,4 @@
     "**/growl": "1.10.2",
     "**/minimist": "^1.2.3"
   }
-}
+}
```

Comment:
It looks like the newline was removed in this file. Can we add that back in here?

Reply:
@theodur if this is a rule we are going enforce should really be a ESLint rule, this is auto-formatting at work.

Reply:
Does not look like we enforce ESLint rules at all...

![image](https://user-images.githubusercontent.com/23741323/141322232-955ea595-d838-4f71-9ff2-dae529af6326.png)

Reply:
Here is a [throwback PR](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/495), we should really enforce our own rules, otherwise what is the point of having them?

Reply:
Didn't realize the rules weren't being enforced in the repo. If we can get that PR working, that'll be great to merge in. I'll take a look it.